src/languages/typescript.js

```diff
@@ -101,14 +101,41 @@ export default function(hljs) {
   const functionDeclaration = tsLanguage.contains.find(m => m.label === "func.def");
   functionDeclaration.relevance = 0; // () => {} is more typical in TypeScript
 
+  const CLASS_REFERENCE = tsLanguage.contains.find(c => c.className === "title.class");
+  const ATTRIBUTE_HIGHLIGHT = tsLanguage.contains.find(c => c.className === "attr");
+
   Object.assign(tsLanguage, {
     name: 'TypeScript',
     aliases: [
       'ts',
       'tsx',
       'mts',
       'cts'
-    ]
+    ],
+    contains: tsLanguage.contains.map(c => {
+      const { contains } = c;
+      if (
+        contains
+      ) {
+        const params = contains.find(cc => cc.scope === 'params' || cc.className === 'params');
```

Comment:
Do you really need params here or just access to params_contains?  See the already exported `tsLanguage.exports.PARAMS_CONTAINS` instead.